front/src/app/app.module.ts

```diff
@@ -3,14 +3,21 @@ import { BrowserModule } from '@angular/platform-browser';
 
 import { AppRoutingModule } from './app-routing.module';
 import { AppComponent } from './app.component';
+import { StyleguideComponent } from './styleguide/styleguide.component';
+import {MatButtonModule} from '@angular/material/button';
+import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
+
 
 @NgModule({
   declarations: [
-    AppComponent
+    AppComponent,
+    StyleguideComponent
   ],
   imports: [
     BrowserModule,
-    AppRoutingModule
+    AppRoutingModule,
+    BrowserAnimationsModule,
+    MatButtonModule,
```

Comment:
Please created a `SharedModule` in `shared` folder and add Angular material dependencies there and then import this module in the module for "styleguide"

You can read about this approach here https://mobinaya.medium.com/shared-module-in-angular-bc56f8264b2e or here  https://dev.to/digitaldino/angular-architecture-shared-module-1mf4